newscoop/library/Newscoop/Tools/Console/Command/AutopublishCommand.php

```diff
@@ -32,9 +33,8 @@ protected function execute(Console\Input\InputInterface $input, Console\Output\O
     {
         $issueActions = \IssuePublish::DoPendingActions();
         $articleActions = \ArticlePublish::DoPendingActions();
-
         if ($issueActions > 0 || $articleActions > 0) {
-            fopen(realpath(APPLICATION_PATH . '/../') .'/reset_cache', 'w');
+            fopen($this->getContainer()->getParameter('kernel.cache_dir') .'/reset_cache', 'w');
```

Comment:
maybe we can provide `$this->getContainer()->getParameter('kernel.cache_dir') .'/reset_cache'` as a parameter ? what do you think ?
```yaml
reset_cache:  '%kernel.cache_dir%/reset_cache'
```

then you will get it like: `$this->getContainer()->getParameter('reset_cache')`